src/datatypes/src/vectors/dictionary.rs

```diff
@@ -71,7 +85,7 @@ impl DictionaryVector {
     }
 }
 
-impl Vector for DictionaryVector {
+impl<K: ArrowDictionaryKeyType> Vector for DictionaryVector<K> {
     fn data_type(&self) -> ConcreteDataType {
         ConcreteDataType::Dictionary(DictionaryType::new(
             ConcreteDataType::int64_datatype(),
```

Comment:
should this be `K`'s type?

Reply:
Fixed in https://github.com/GreptimeTeam/greptimedb/pull/6855/commits/615bceb3d014cbb0d54185f068c0e1e442f688d1